packages/mobile/src/geth/saga.ts

```diff
@@ -127,19 +126,22 @@ function* waitForGethInstance() {
       case FailedToFetchGenesisBlockError:
         return GethInitOutcomes.NETWORK_ERROR_FETCHING_GENESIS_BLOCK
       default: {
+        throw error
```

Comment:
Did you mean to throw here? Makes the net two lines unreachable

Reply:
Good catch - debugging leftover